all.bash

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+bash checkformat.bash
+bash vet.bash
```

Comment:
Maybe combine `checkformat.bash` and `vet.bash` into `check.bash`? Probably should be invoked after build.bash.